misc/navigation.py

```diff
@@ -3,6 +3,40 @@
 ctx = Context("navigation")
 
 keymap = {
+    # Requires activation of System Preferences -> Shortcuts -> Input Sources
+    # -> "Select the previous input source"
+    "change language": Key("ctrl-space"),
+    # Application navigation
+    "swick": Key("cmd-tab"),
+    "new window": Key("cmd-n"),
+    "(next window | gibby)": Key("cmd-`"),
+    "(last window | shibby)": Key("cmd-shift-`"),
+"next space": Key("cmd-alt-ctrl-right"),
+    "last space": Key("cmd-alt-ctrl-left"),
+    # deleting
+    # '(delete line)': Key('cmd-right cmd-backspace'),
+    "steffi": Key("alt-ctrl-backspace"),
+    "stippy": Key("alt-ctrl-delete"),
+    "carmex": Key("alt-backspace"),
```

Comment:
many of these commands are already in `generic_editor`. The ones that are not should potentially go into a new `voicecode.py`, though i'm not sure how many will actually be left.

Reply:
With regards to the navigation file, I thought I was up-to-date with the main repository. Should I go through the navigation file and cross-reference the files you have mentioned to remove repetition?

Reply:
there was a refactor recently that might be the cause of these duplicates. Would be ideal to go through and remove any that already exist elsewhere though. Otherwise it is hard for people to disable/change them (must disable/change them in two places). Sorry its so much work.

Reply:
It's okay. I will do that.

Reply:
I believe I have removed all of the duplication from the navigation file. Let me know if this is sufficient.

Reply:
:+1: